Dockerfile.worker

```diff
@@ -36,7 +36,5 @@ WORKDIR /
 
 COPY --from=builder /workspace/worker /worker
 RUN mkdir -p /mnt/img
-RUN chown 65532:65532 /mnt/img
```

Comment:
Don't we need this?

Reply:
No, since worker is not running under User 65532, then we don't need it

Reply:
What user is it running as then?

Reply:
the default one - "0"